### DISCUSSION TITLE
Support for AWS S3 Event Notifications

### DISCUSSION THREAD

**Discussion by sboeckelmann** [Seagate/cortx-s3server#1393] [General] [answered]
Does cortx-s3server also support S3 Event Notifications or does it only provide a S3 compatible RESTful API?
I am especially interested in s3:ObjectCreated and s3:ObjectRemoved events.

**Comment by johnbent** [comment, marked as answer]
Hello @sboeckelmann . Thanks for starting this discussion. And you're asking at a great time! cortx-s3server does not current support event notifications . . . but @andriytk and @siningwuseagate are about to start adding this next month. If you want to track that work, they will be using the internal [FDMI functionality](https://github.com/Seagate/cortx-motr/tree/main/fdmi) within the [motr component](https://github.com/Seagate/cortx-motr) to detect these events and then will use webhooks(?) to deliver the notifications. Some of that work will be done in the motr repo and some will be done in the [multisite repo](https://github.com/Seagate/cortx-multisite). Please feel free to "watch" the multisite repo if you'd like to follow along or contact Andriy and Sining if you'd like to be even more involved.

Also, when this feature is complete, @novium258 will be announcing it in our monthly newsletter so that would be another way for you to get the "Event Notification" when this feature is completed. You can see up for the newsletter [here ](Cortx.link/newsletter)if you would like.
